.github/workflows/azure_kubernetes_aks_single_region_tests.yml

```diff
@@ -0,0 +1,588 @@
+---
+name: Tests - Integration - Azure Kubernetes AKS Single Region
+
+permissions:
+    id-token: write
+    contents: read
+
+on:
+    schedule:
+        - cron: 0 3 * * 3 # Runs at 3 AM on Wednesdays
+    pull_request:
+        paths:
+            - .github/workflows/azure_kubernetes_aks_single_region_tests.yml
+            - .tool-versions
+            - generic/kubernetes/single-region/**
+            - azure/kubernetes/aks-single-region/**
+            - azure/modules/**
+            - generic/kubernetes/single-region/procedure/**
+            - .github/actions/azure-kubernetes-aks-single-region-create/**
+            - .github/actions/azure-kubernetes-aks-single-region-cleanup/**
+
+    workflow_dispatch:
+        inputs:
+            delete_clusters:
+                description: Whether to delete the clusters.
+                type: boolean
+                default: true
+            enable_tests:
+                description: Whether to enable the tests.
+                type: boolean
+                default: true
+            cluster_name:
+                description: Name of the cluster to deploy.
+                type: string
+                required: false
+            resource-group-name:
+                description: Name of the resource group.
+                type: string
+                required: false
+            resource-prefix:
+                description: Prefix for the resource names.
+                type: string
+                required: false
```

Comment:
maybe we could add a default for this as in theory it's sufficient to have this.

+ add a description indicating that either it's prefix or the other two.

Reply:
or alternatively remove the input option about the prefix to not confuse.